pydantic/env_settings.py

```diff
@@ -56,8 +60,21 @@ def _build_environ(self) -> Dict[str, Optional[str]]:
             d[field.alias] = env_val
         return d
 
+    def _read_from_env_file(self, file_path: str) -> Dict[str, str]:
+        file_vars = {}  # type: Dict[str, str]
+        with open(file_path) as env_file:
+            for line in env_file.readlines():
```

Comment:
probably easier to just do `enumerate(file_path.read_text().split('\n'), start=1)` (tracking line number for errors below).

Reply:
I was under the impression that `.readlines()` is better for cross-platform support than using `.split()`.

Reply:
maybe, alternatively we could use `re.split('[\r\n]+', file_path.read_text())` which avoids keeping the file open while we process lines and I guess should be a little faster.

Don't really mind.